index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <p>Alejandro Ramos</p>
+        <img src="images/IconMenu.png" alt="Three horizontal dashes button">
+    </header>
+    <main class="mainContain">
+        <div class="informativeText">
+            <h1>Hey There.<br>I'm Alex</h1>
+            <h2>I’am a Software Developer</h2>
+            <p>I can help you build a product, feature or<br>website Look through some of my work<br>and experience! If you like what you see<br>and have a project you need coded,<br> 
+            don’t hestiate to contact me.</p>
+        </div>
+        <div class="socialIcons">
+             <img src="images/IconGitHub.png" alt="-GitHub button">
+             <img src="images/IconLinkedin.png" alt="Linkedin button">
+             <img src="images/IconAngelist.png" alt="Angelist button">
+             <img src="images/IconTwitter.png" alt="Twitter button">
+             <img src="images/IconMedium.png" alt="Medium button">
+        </div>        
+    </main>
```

Comment:
- [ ] Kindly let's use the `<section>` tag as the container of the headline section instead of the `<main>`, since `<main>` is instead used to wrap all the sections so we can not take it as the section itself 👍 
- [ ] It could be a good idea to have the social media icons as links, this is because we will need to click on them for us to be taken to that specific profile, and at the end, we can group them together using an unordered list `<ul>`, At the end, it should look something like this 👇 
```
<ul>
     <li>
        <a href="URL to your profile of the social media">
            <img src="images/IconGitHub.png" alt="-GitHub button">
        </a>
     </li>
      <li>
        <a href="URL to your profile of the social media">
            <img src="images/IconLinkedin.png" alt="Linkedin button">
        </a>
     </li>
</ul>
```
With this, our code will be cleaner and semantic, I promise 😉

Reply:
Thanks for teaching me, Baraka, your promise was right 😄